pkg/common/telemetry/common/label_adders.go

```diff
@@ -15,8 +15,10 @@ func AddAttestorType(cc *telemetry.CallCounter, aType string) {
 }
 
 // AddAudience add the Audience label(s) to the given counter
-// from the given audience(s)
+// from the given audience(s).
 func AddAudience(cc *telemetry.CallCounter, auds ...string) {
+	// the number of audiences should be small in a secure system, so we label every audience
```

Comment:
Where is this used? This looks like it will cause issues in a statsd environment. We can't be variable about what the item at "position X" is, this looks like it will cause that.

Reply:
didn't think of this with your other comment; removing